corehq/apps/app_manager/templates/app_manager/partials/modules/module_actions.html

```diff
@@ -14,15 +14,16 @@
           Please select the case lists/detail menu(s) that needs to be overwritten.
         {% endblocktrans %}
     </p>
-    <p>
-      <a class="btn btn-primary-dark"
-         target="_blank"
-         href="https://confluence.dimagi.com/display/commcarepublic/Overwriting+Case+List+and+Case+Detail+Configuration">
-        {% trans "Learn More" %}
-      </a>
-    </p>
   </div>
 {% endif %}
+<div>
+  <p>{% trans 'Overwritting Case List and Case Detail Configuration.' %}
```

Comment:
```suggestion
  <p>{% trans 'To save time when building large applications, it can be useful to copy one menu's entire case list or case details configuration to another menu.' %}
```

Reply:
Usually intro text on a page is phrased as sentences. I just grabbed this wording from the docs.